packages/core/src/interpreter.ts

```diff
@@ -270,6 +279,36 @@ export class Interpreter<
     };
   }
 
+  public inspect(
+    observer: Observer<InternalStateFrom<TBehavior>>
+  ): Subscription;
+  public inspect(
+    nextListener?: (state: InternalStateFrom<TBehavior>) => void,
+    errorListener?: (error: any) => void,
+    completeListener?: () => void
+  ): Subscription;
+  public inspect(
```

Comment:
Like we discussed before - I think this should not listen for (internal) states but rather for tailored events that would inform the inspect about various things that happen to the actor. I assume this is a draft right now, and we are waiting on Gavin's to finalize the "protocol" part for this?